.pre-commit-config.yaml

```
@@ -0,0 +1,46 @@
+  - repo: https://github.com/sirosen/texthooks
+    rev: 0.7.1
+    hooks:
+      - id: alphabetize-codeowners
+      - id: fix-smartquotes
+
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v4.0.0-alpha.8
+    hooks:
+      - id: prettier
+        types_or: [yaml, markdown, json]
+        exclude: go.sum 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)